### PR TITLE
tests: Log master ip addresses while checking for cluster bootstrapping

### PR DESCRIPTION
**Pull request by mxinden**
In the loop of checking masters to complete the bootstrapping process,
this patch adds a simple log of the master ip addresses that are being
checked. In addition it removes the final `elapsed` reference which was
not defined in that scope.

@kalmog Thanks for pointing this out to me.

### CODE REVIEW

File: tests/rspec/lib/cluster.rb [hunk 1]
```diff
@@ -145,16 +145,21 @@ def wait_for_bootstrapping
...
     from = Time.now
+    ips = []
+
     180.times do # 180 * 10 = 1800 seconds = 30 minutes
       ips = master_ip_addresses
       return if service_finished_bootstrapping?(ips, service)
 
       elapsed = Time.now - from
-      puts "Waiting for bootstrapping of #{service} service to complete..." if (elapsed.round % 5).zero?
+      if (elapsed.round % 5).zero?
```
Comment (cpanato) [inline tests/rspec/lib/cluster.rb]: you can do the same and still keep the onliner
`puts "Waiting for bootstrapping of #{service} service to complete...\nChecked master nodes: #{ips}" if (elapsed.round % 5).zero?`

just a comment dont need to implement if you think it is not useful

Comment (mxinden) [reply tests/rspec/lib/cluster.rb]: True, just preferred the shorter lines for easier readability.

**Comment by mxinden** [comment]
@cpanato Thanks for the quick review!

**Comment by mxinden** [comment]
Ok to test

**Comment by mxinden** [comment]
ok to test

**Comment by squat** [comment]
ok to test

**Comment by cpanato** [comment]
retest this please

**Comment by cpanato** [comment]
retest this please

**Pull request merged**